### ISSUE TITLE
ServiceDnsName in Application Manifest not parameterizable

### ISSUE THREAD

**Issue by aelij** [Azure/service-fabric-issues#540]
For example:

```xml
<Service Name="Stateless1" ServiceDnsName="[Stateless1DnsName]">
```

Produces the deployment error:
> The ServiceDnsName for DefaultService 'Stateless1' is invalid.

This essentially prevents us from creating multiple named instances of the same application type (using default services).

**Comment by wminos** [comment]
This issue is probably a minor issue and there are ways to solve it in different ways.
However, I hope that if this issue is resolved, convenience development through VS2017 will be improved.

**Comment by ninzavivek** [comment]
The fix for this should be available in  SF 6.2 CU1.

**Comment by chunliu** [comment]
@ninzavivek can you confirm if your fix is in SF 6.2 CU1? I installed the runtime version 6.2.274.9494 and SDK version 3.1.274, but I still hit this problem. 

`2>Test-ServiceFabricApplicationPackage : The ServiceDnsName for DefaultService 'db.todo' is invalid.`

Do I need to change anything in my existing project?
